skl2onnx/operator_converters/TextVectorizer.py

```diff
@@ -52,7 +52,7 @@ def _intelligent_split(text, op, tokenizer, existing):
     if spl in existing:
         raise RuntimeError("The converter cannot guess how to "
                            "split an expression into tokens.")
-    if op.ngram_range[0] == 1:
+    if op.ngram_range[0] == 1 and op.ngram_range[1] > 1:
```

Comment:
do we need to check len(op.ngram_range) > 1 before check the its value?

Reply:
len(ngram_range) will always be 2 in a fit model.